lib/SILOptimizer/Analysis/EscapeAnalysis.cpp

```diff
@@ -449,6 +470,10 @@ void EscapeAnalysis::ConnectionGraph::computeUsePoints() {
       for (CGNode *Def : Node->defersTo) {
         Changed |= Def->mergeUsePoints(Node);
       }
+      for (Predecessor Pred : Node->Preds) {
+        if (Pred.getInt() == EdgeType::Defer)
+          Changed |= Pred.getPointer()->mergeUsePoints(Node);
+      }
```

Comment:
Propagating in both defer directions makes sense. This also seems like an existing correctness issue.

Reply:
No, again. Not propagating in reverse direction of defer edges was worked-around by also checking the content node (all nodes in a defer-web have the same content node)